### PR TITLE
Vscode Shortcuts

### PR DESCRIPTION
**Pull request by Xuzzo**
Add default vscode keyboard shortcuts in oop-essentials readme

### CODE REVIEW

File: oop-essentials/ide-features/README.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ When jumping between files, being able to quickly return to a previous file (wit
 | IDE       | Shortcut       |
 | --------- |----------------|
 | JetBrains | Ctrl+E (Cmd+E) |
-| VSCode    | TODO           |
+| VSCode    | Ctrl+Alt+Left/Right (Cmd+Opt+Left/Right) |
```
Comment (opcode81) [inline oop-essentials/ide-features/README.md:143]: This is probably some other navigation. I believe the recent editors are also Ctrl+E in VSCode?

File: oop-essentials/ide-features/README.md [hunk 2]
```diff
@@ -157,14 +157,14 @@ Auto-completion helps save time by not requiring the developer to type out the n
...
-| VSCode    | TODO |
+| VSCode    | just start typing |
 
 **Auto-completion of identifiers in the entire project**, including external dependencies:
 
 | IDE       | Shortcut  |
 | --------- | --------- |
 | JetBrains | Ctrl+Space+Space |
-| VSCode    | TODO |
+| VSCode    | just start typing |
```
Comment (opcode81) [inline oop-essentials/ide-features/README.md:167]: Are you sure this works? In this case, we are trying to find an identifier which has not yet been imported in the current model and which could be anywhere (in the same project or its dependencies)!

Comment (opcode81) [reply oop-essentials/ide-features/README.md:167]: I just verified that it works for in-project identifiers but not for dependencies.
Any way to find and auto-import, for example, StandardScaler from sklearn?

File: oop-essentials/ide-features/README.md (hunk 1 above)
Comment (Xuzzo) [reply oop-essentials/ide-features/README.md:143]: ah damn yes this is to navigate back and forth to files. cmd+P allows you to search and see all the recently used files. Let me change this

File: oop-essentials/ide-features/README.md (hunk 2 above)
Comment (Xuzzo) [reply oop-essentials/ide-features/README.md:167]: Mmmh I do not know of anything working for dependencies...

Comment (Xuzzo) [reply oop-essentials/ide-features/README.md:167]: ah wait, you need to activate the python>Analysis: Auto Import Completions in settings

**Pull request merged**